packages/server-wallet/src/engine/__test__/fixtures/test-channel.ts

```
@@ -199,15 +199,25 @@ export class TestChannel {
   }
 
   public get openChannelObjective(): OpenChannel {
-    return {
-      participants: this.participants,
-      type: 'OpenChannel',
-      data: {
-        targetChannelId: this.channelId,
-        fundingStrategy: this.fundingStrategy,
-        fundingLedgerChannelId: this.fundingLedgerChannelId,
-      },
-    };
+    const {participants, fundingStrategy, channelId: targetChannelId} = this;
+    const type = 'OpenChannel' as const;
+    let data: OpenChannel['data'];
+    switch (fundingStrategy) {
+      case 'Ledger': {
+        const fundingLedgerChannelId = this.fundingLedgerChannelId as string;
```

Maybe throw an error if `this.fundingLedgerChannelId` is undefined?